mozjs/js/rust/tests/magic_dom_impl.rs

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#[macro_use] extern crate magic_codegen;
+extern crate js;
+
+use js::jsapi;
+use js::magic::{MagicSlot, SlotIndex};
+use js::rust::{GCMethods, RootKind};
+
+use std::mem;
+use std::ptr;
+
+#[allow(dead_code)]
```

Comment:
What's causing the dead code? Is it just that the code generated from the derive isn't marked `pub`?